easybuild/tools/config.py

```diff
@@ -517,46 +521,3 @@ def module_classes():
 def read_environment(env_vars, strict=False):
     """NO LONGER SUPPORTED: use read_environment from easybuild.tools.environment instead"""
     _log.nosupport("read_environment has moved to easybuild.tools.environment", '2.0')
-
-
-def set_tmpdir(tmpdir=None, raise_error=False):
```

Comment:
this is internal to framework, if someone is using this in an easyblock, they're doing something weird or something they shouldn't be doing ;)